File: help/eng_gb/exercicios_da_ficha_9/Ficha09_008/Ficha09_008.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha09_008 {

    //Main Function
    static void Main() {
        Console.Write("Fibonaci sequence sum\n " );
        int f0 = 1 ;
        int f1 = 1 ;
        Console.Write(f0 + " " + f1 );
        int sum = 2 ;
        while(f0 + f1 < 100 ){
            int fn = f0 + f1 ;
            sum +=fn;
            Console.Write(" " + fn );
            f1 = f0 ;
            f0 = fn ;
        }
        Console.Write("\nSum = " + sum );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:11

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_008/Ficha09_008.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Fibonaci sequence sum\n " );
    var f0 = 1 ;
    var f1 = 1 ;
    document.write(f0 + " " + f1 );
    var sum = 2 ;
    while(f0 + f1 < 100 ){
        var fn = f0 + f1 ;
        sum +=fn;
        document.write(" " + fn );
        f1 = f0 ;
        f0 = fn ;
    }
    document.write("\nSum = " + sum );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:11
